logging/simplelog/simplelog.go

```diff
@@ -0,0 +1,134 @@
+// iample simplelog writes some entries, lists them, then deletes the log.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	// [START imports]
+	// NOTE: This will become cloud.google.com/go/logging soon.
+	"cloud.google.com/go/preview/logging"
+
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	// [END imports]
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: simplelog <project-id>")
+		os.Exit(2)
+	}
+	projID := os.Args[1]
+
+	// [START setup]
+	ctx := context.Background()
+	client, err := logging.NewClient(ctx, projID,
+		// Admin scope is required to delete logs.
+		option.WithScopes(logging.AdminScope))
+
+	if err != nil {
+		log.Fatalf("Failed to create logging client: %v", err)
+	}
+	client.OnError = func(err error) {
+		// Print an error to the local log.
+		// For example, if Flush() failed.
+		log.Printf("client.OnError: %v", err)
+	}
+	// [END setup]
+
+	log.Print("Writing some log entries.")
+	writeEntry(client)
+	structuredWrite(client)
+
+	// It takes some time for the log entries to appear.
+	time.Sleep(5 * time.Second)
+
+	log.Print("Fetching and printing log entries.")
+	entries, err := getEntries(client, projID)
+	if err != nil {
+		log.Fatalf("Could not get entries: %v", err)
+	}
+	log.Printf("Found %d entries.", len(entries))
+	for _, entry := range entries {
+		fmt.Printf("Entry: %6s @%s: %v\n",
+			entry.Severity,
+			entry.Timestamp.Format(time.RFC3339),
+			entry.Payload)
+	}
+
+	log.Print("Deleting log.")
+	if err := deleteLog(client); err != nil {
+		log.Fatalf("Could not delete log: %v", err)
+	}
+}
+
+func writeEntry(client *logging.Client) {
+	// [START write_log_entry]
+	const name = "log-example"
+	logger := client.Logger(name)
+	defer logger.Flush() // Ensure the entry is written.
+
+	stdlog := logger.StandardLogger(logging.Info)
+	stdlog.Printf("stdlog is a standard library log.Logger with INFO severity.")
+	// [END write_log_entry]
+}
+
+func structuredWrite(client *logging.Client) {
+	// [START write_structured_log_entry]
+	const name = "log-example"
+	logger := client.Logger(name)
+	defer logger.Flush() // Ensure the entry is written.
+
+	logger.Log(logging.Entry{
+		// Log anything that can be marshaled to JSON.
+		Payload: struct{ Anything string }{
+			Anything: "The payload can be any type!",
+		},
+		Severity: logging.Debug,
+	})
+	// [END write_log_entry]
+}
+
+func deleteLog(client *logging.Client) error {
+	ctx := context.Background()
+
+	// [START delete_log]
+	const name = "log-example"
+	if err := client.DeleteLog(ctx, name); err != nil {
+		return err
```

Comment:
It's for a snippet, so I thought `return err` might be better without any context. What do you think?